rmw_connext_cpp/src/functions.cpp

```diff
@@ -174,6 +174,23 @@ struct ConnextStaticServiceInfo
   const service_type_support_callbacks_t * callbacks_;
 };
 
+rmw_ret_t rmw_check_attach_condition_error(DDS_ReturnCode_t retcode)
```

Comment:
Since this is not a function specified in the rmw interface please use a different name (e.g. without the `rmw_` prefix).

Same below and for OpenSplice.